input/otlp/metrics.go

```diff
@@ -48,30 +48,71 @@ import (
 	"github.com/elastic/apm-data/model/modelpb"
 )
 
+type ConsumeMetricsResult struct {
+	RejectedMetrics    int64
```

Comment:
Do we need to track Rejected/AcceptedMetrics in addition to data points? I don't think this ends up in the response to the client.

Reply:
RejectedMetrics is needed to not break existing monitoring. We have [monitoring in apm-server](https://github.com/elastic/apm-server/blob/main/internal/beater/otlp/common.go#L59) for unsupported metrics (not data points). Line 79 is responsible for recording RejectedMetrics.

Reply:
OK, I see that we're reporting it through `Stats()`, so we need to measure it. Does it need to be returned in the result of `ConsumeMetrics` though?